account_financial_report/report/general_ledger.py

```diff
@@ -550,7 +615,7 @@ def _inject_account_values(self):
                 tuple(self.filter_cost_center_ids.ids),
             )
         query_inject_account_params += (
-            self.id,
+            self.id or 'NULL',
```

Comment:
Why do you add the `or 'NULL'`?
The report_id must not is null.
We need the `report_id` to distinct each different asking of reports.